### DISCUSSION TITLE
Add a nice documentation why and how to set the DisplayVersion for Application developers

### DISCUSSION THREAD

**Discussion by Yolgie** [microsoft/winget-pkgs#41881] [Ideas]
Hi, 

I've contacted a few developers (smaller like stickies & f.lux and bigger like gog.galaxy) asking them politely to include the DisplayVersion in their application, explaining it in similar words as Levvie here https://github.com/microsoft/winget-pkgs/issues/27206#issuecomment-915047667 but I have not found a good source to redirect them to if they need/want more information. Is there something like a documentation from Microsoft detailing that DisplayVersion should be written (and why) or at least from winget? If not, maybe this would help in the effort to get developers to adapt their installers :)

**Comment by denelon** [comment]
@Yolgie,

That is a great question. I'm asking the various teams if we have a "best-practices" set of documents. This is likely to vary by the installer technology. For example: MSIX, MSI, and the different .exe installer technologies would likely have different methodologies to ensure the version is reported properly in the registry.

**Comment by JohnMcPMS** [comment]
This is more about the user experience, but I think that maybe somewhere close to this kind of documentation could make sense:
https://docs.microsoft.com/en-us/windows/win32/uxguide/exper-setup

Alternately, we could have winget specific documentation about what information they can write to ARP that makes things work.
